content/assignments/hello-world/_index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Assignment 0: Hello, World!"
+date: 2023-01-05T09:57:32-06:00
+draft: false
+weight: 100
+originalAuthor: Courtney  # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+### Table of Contents
+1. Getting Started
+1. Running the Auto-graded Tests
+1. Your Task
+1. Submitting Your Work
+
+Use this assignment as a tutorial for running tests in Visual Studio, working in C#, and submitting to GitHub Classroom. Parts 1 and 2 will help you get familiar with both C# and Visual Studio.  Part 3 is where you will code.  Part 4 reviews the submission process.
+
+## Getting Started
+Let’s set up the assignment on our computer and learn about its basic structure.
+
+<!-- TODO: Update Link for Canvas-->
```

Comment:
Wouldn't this change for each class?

Reply:
Yes, and that would be terrible.  I'll format it like the sphinx book.  Students will find the link to GHClassroom in their class canvas as they currently do now.